src/main/java/com/training/tinyurl/controller/TinyUrlControllerImpl.java

```diff
@@ -29,11 +30,18 @@ public TinyUrlControllerImpl(ITinyUrlService tinyUrlService) {
     @Override
     public ResponseEntity<String> registerUser(@RequestBody @Valid RegistrationReqDto request,
                                                BindingResult bindingResult)
-            throws ValidationException, MongoApiException {
+                                                throws ValidationException, MongoApiException {
         Validator.validate(bindingResult);
         tinyUrlService.createNewUser(request);
         log.info(request.getEmail()+" successfully registered");
         return new ResponseEntity<>(request.getEmail()+" successfully registered",HttpStatus.CREATED);
     }
 
+    @GetMapping("login")
+    @Override
+    public ResponseEntity<String> loginUser() {
+        log.info("Login successful");
```

Comment:
log username for which login is successful.
This log message will be a matchstick in the sea aka garbage text when in actual production millions of users log in.